dgraph/src/jepsen/dgraph/trace.clj

```diff
@@ -3,30 +3,48 @@
                                 Tracing
                                 Span)
            (io.opencensus.trace.samplers Samplers)
-           (io.opencensus.exporter.trace.logging LoggingTraceExporter)))
+           (io.opencensus.exporter.trace.logging LoggingTraceExporter)
+           (io.opencensus.exporter.trace.jaeger JaegerTraceExporter)))
 
-(def tracer ^Tracer (Tracing/getTracer))
-(def trace-exporter (LoggingTraceExporter/register))
+(def tracer         (atom nil))
+(def trace-exporter (atom nil))
+(def sampler        (atom nil))
+(def trace-config   (atom nil))
```

Comment:
I think I missed this earlier, but it feels like something that might belong on the test map, rather than stateful globals? Though if this is just a one-off hack for this test, I don't feel too strongly about it. Thoughts? :)